src/http.ts

```diff
@@ -195,6 +195,25 @@ export const POST = {
       throw new Error(`Server response: "${response.status}"`);
     }
   },
+  qmi: async (
+    qmiId: number, action: 'start' | 'stop', payload: any = undefined,
+  ): Promise<request.Response> => {
+    const headers: { [key: string]: string } = {
+      'X-User-Id': config.userToken,
+    };
+    const options = {
```

Comment:
Any reason you're not using `http._post(...)`?

Reply:
This returns 204 for success - `http._post` throws an error in that case.

Reply:
That's a good callout. This is a limitation of the `_request` function in that it should accept any `2xx` response code, not just `201` and `202`. I'd prefer that you modify that function to accommodate your use here. Can we get that chance into this PR?

Reply:
I will do this in a follow up PR to make sure we're not masking any existing error messages.